psi4/driver/driver_findif.py

```diff
@@ -0,0 +1,798 @@
+from __future__ import division
```

Comment:
Add license header and delete the future import (can be the first py2-free file).